### PR TITLE
feat: create music genre details page

### PR DESCRIPTION
**Pull request by jacob-partner**
d23139f Add new page for music details.

### CODE REVIEW

**Comment by jacob-partner** [comment]
@Alfonso4V @marcoLopezAIA Me gustaría obtener retroalimentación antes de hacer el Merge.

File: src/genre.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Música</title>
+  </head>
+  <style></style>
```
Comment (Alfonso4V) [inline src/genre.html:20]: Esta etiqueta sobra, ya que no contiene nada.

Comment (jacob-partner) [reply src/genre.html:20]: Thanks, I'll check it out

**Comment by marcoLopezAIA** [comment]
I need a doc or a .md doc showing all the commands you used in the session :) pls

File: src/genre.html (hunk 1 above)
Comment (jacob-partner) [reply src/genre.html]: Ready!

**Comment by jacob-partner** [comment]
> I need a doc or a .md doc showing all the commands you used in the session :) pls

This is a feature branch. Create an issue and I'll work on it on a different branch.

**Review by marcoLopezAIA — APPROVED** [review]

**Pull request merged**